### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by cordeliachen**

### CODE REVIEW

File: students/cc4655.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ GitHub README: [link](https://github.com/cordeliachen/cordeliachen/blob/main/REA
 
 ## Project Proposal
 
-I would like to make a tool to directly download music from Youtube and import it into your local music library.
+- [spotify-charts](./projects/python/spotify-charts.md)
```
Comment (timkpaine) [inline students/cc4655.md:13]: update link path and re-squash 😈

File: students/cc4655.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ GitHub README: [link](https://github.com/cordeliachen/cordeliachen/blob/main/REA
...
+- [spotify-charts repo](https://github.com/cordeliachen/spotify-charts)
 
 ## Project Contributions
 
-I want to contribute to chartify, possibly by implementing a new feature resuested [here](https://github.com/spotify/chartify/issues/136).
+- [librosa](./projects/python/librosa.md)
```
Comment (timkpaine) [inline students/cc4655.md:18]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/python/spotify-charts.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# spotify-charts
+
```
Comment (timkpaine) [inline projects/python/spotify-charts.md:2]: add link to your repo

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/cc4655.md (hunk 2 above)
Comment (cordeliachen) [reply students/cc4655.md]: done

File: projects/python/spotify-charts.md (hunk 3 above)
Comment (cordeliachen) [reply projects/python/spotify-charts.md:2]: sorry, thought you meant the uni file!

File: students/cc4655.md (hunk 1 above)
Comment (cordeliachen) [reply students/cc4655.md]: done

File: students/cc4655.md (hunk 2 above)
Comment (cordeliachen) [reply students/cc4655.md]: done

**Review by timkpaine — APPROVED** [review]
lgtm

**Pull request merged**